pytest-selenium/utils/utility.py

```diff
@@ -3,6 +3,7 @@
 from __future__ import annotations
 
 from enum import Enum
+import json
```

Comment:
switch the import from the full package to just the `loads` function (`from json import loads`), unless you plan to use more from the `json` library (then the import should be moved above `enum` as full packages should be before select function imports (`from ... import ...` )

Reply:
@gregfitch updated